pkg/controller/workload/workload_processor.go

```diff
@@ -587,10 +603,46 @@ func (p *Processor) handleAddressTypeResponse(rsp *service_discovery_v3.DeltaDis
 	}
 
 	_ = p.handleRemovedAddresses(rsp.RemovedResources)
+	p.compareWorkloadAndServiceWithHashName()
 
 	return err
 }
 
+// When processing the workload's response for the first time,
+// fetch the data from the /mnt/workload_hash_name.yaml file
+// and compare it with the data in the cache.
+func (p *Processor) compareWorkloadAndServiceWithHashName() {
+	var (
+		bk = bpf.BackendKey{}
+		bv = bpf.BackendValue{}
+	)
+
+	if kmeshbpf.GetStartType() == kmeshbpf.Normal {
```

Comment:
Since we haven't support upgrade, to be safer shall we compare with `RESTART`  only?

Reply:
yes, i will change it